SourceX/miniwin/misc_dx.cpp

```diff
@@ -24,6 +24,8 @@ WINBOOL SetCursorPos(int X, int Y)
 #endif
 
 	SDL_WarpMouseInWindow(window, X, Y);
+	MouseX = X;

```

Comment:
SDL_WarpMouseInWindow already causes a mouse move event which updates MouseX/Y, is something not working correctly with this?

Reply:
Perhaps sometimes `plrctrls` needs an immediate update, before the event is processed. Not sure if this is still the case though.

Reply:
Stuff started not working when removing this one, most others appeared to be fine to remove.

Reply:
Removing this fixed upscalling and i'm not seeing any issues, go figure.